package.json

```diff
@@ -5,11 +5,12 @@
   "repository": "https://github.com/quran/quran.com-frontend",
   "scripts": {
     "test": "npm run test:dev:unit",
-    "test:ci:unit": "karma start --browsers PhantomJS --single-run; npm run test:ci:lint",
+    "test:ci:unit": "karma start --browsers PhantomJS --single-run",
+    "posttest:ci:unit": "npm run test:ci:lint",
     "test:ci:functional": "BROWSER=phantomjs ./tests/functional/test.sh start-ci",
     "posttest:ci:functional": "./tests/functional/test.sh stop",
     "test:dev:unit": "karma start",
-    "test:ci:lint": "eslint ./src/**/*.js",
+    "test:ci:lint": "./node_modules/eslint/bin/eslint.js ./src/**/*.js",
```

Comment:
hmm, i don't think that was the case with my local machine... it was not reflecting that since my Eslint installed globally was not the same version as package.json has.